src/event-target.ts

```diff
@@ -0,0 +1,29 @@
+/**
+ * Extend EventTarget to include the Transcend toStringTag
+ */
+export class TranscendEventTarget extends EventTarget {
+  #stringTag = 'Transcend';
+
+  /**
+   * Add Transcend to the string tag
+   *
+   * @returns string
+   */
+  get [Symbol.toStringTag](): string {
+    return this.#stringTag;
+  }
+
+  /**
+   * Allow airgap.js code to overwrite [Symbol.toStringTag]
+   */
+  set [Symbol.toStringTag](updatedStringTag: string) {
+    this.#stringTag = updatedStringTag;
+  }
+
+  /**
+   * @param args - EventTarget constructor parameters
+   */
+  constructor(...args: ConstructorParameters<typeof EventTarget>) {
+    super(...args);
+  }
+}
```

Comment:
@eligrey here's the solve for your suggestion (which i realized is actually necessary since airgap will try to write the toStringTag and it will throw because the EventTarget property is readonly)

Reply:
This looks fine although I can simplify it some more